locales/en.json

```diff
@@ -77,5 +77,7 @@
 	"Requested updated successfully": "Requested updated successfully",
 	"Request is already APPROVED!": "Request is already APPROVED!",
 	"Request deleted successfully!": "Request deleted successfully!",
-	"Only PENDING request can be edited!": "Only PENDING request can be edited!"
+	"Only PENDING request can be edited!": "Only PENDING request can be edited!",
+	"No request found!": "No request found!",
+	"Manager Id does not exist.": "Manager Id does not exist."
```

Comment:
You can just add the French ,Swahili & Kinyarwanda versions for those messages.
to simplify it, for `No request found!` you can choose to use `No request found.` because that already exist. Anyways you did an awesome task, thanks

Reply:
This is not my work , I only worked on notifications